### PR TITLE
Update 163.com-mixedcontent.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/163.com-mixedcontent.xml [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
+	<target host="chat.study.163.com" />
 	<target host="email.163.com" />
 	<target host="mail.163.com" />
 	<target host="i.money.163.com" />
 	<target host="piao.163.com" />
 	<target host="trip.163.com" />
-
-
+	
+	<target host="ct.stu.126.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/163.com-mixedcontent.xml]: why is this mixed content?

Comment (renyouguo) [inline src/chrome/content/rules/163.com-mixedcontent.xml]: It comes from chat.study.163.com .
Should I open another PR for it?

Comment (J0WI) [inline src/chrome/content/rules/163.com-mixedcontent.xml]: You can include it into 163.com.xml since this hosts servs no mixed content.

**Comment by renyouguo** [comment]
#4755

**Pull request closed without merging**